### PR TITLE
Added decimal_isolate.py

### PR DESCRIPTION
**Pull request by JakeGerber**
### Created a function program that isolates the decimal from a number.

* [ X] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [X ] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [ X] This pull request is all my own work -- I have not plagiarized.
* [ X] I know that pull requests will not be merged if they fail the automated tests.
* [X ] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [X ] All new Python files are placed inside an existing directory.
* [ X] All filenames are in all lowercase characters with no spaces or dashes.
* [X ] All functions and variable names follow Python naming conventions.
* [ X] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [ X] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [ X] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [X ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by xcodz-dot** [comment]
run this command on your pc:
`pip install black`
then run this:
`black decimal_isolate.py`
then upload the file changed by black

**Comment by JakeGerber** [comment]
Thank you so much for the help!

**Comment by xcodz-dot** [comment]
Your Welcome!

**Comment by xcodz-dot** [comment]
@cclauss, check this out

**Comment by cclauss** [comment]
@xcodz-dot You know how this works... Needs doctests and type hints as discussed in CONTRIBUTING.md.

File: maths/decimal_isolate.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+    """
+    Isolates the decimal part of a number.
+    If digitAmount > 0 round to that decimal place, else print the entire decimal.
+    >>> decimal_isolate(35.345, 1)
+    0.3
+    >>> decimal_isolate(35.345, 2)
+    0.34
+    >>> decimal_isolate(35.345, 3)
+    0.345"""
```
Comment (cclauss) [inline maths/decimal_isolate.py:16]:
```suggestion
    0.345
    """
```

File: maths/decimal_isolate.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+"""
+Isolate the Decimal part of a Number
+"""
```
Comment (cclauss) [inline maths/decimal_isolate.py:3]: Please provide a URL to a document describing this algorithm and why it is useful.

**Comment by cclauss** [comment]
Why not just use mod (num % 1) ?

**Comment by cclauss** [comment]
Please edit the commit message to be a https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/about-task-lists

**Comment by JakeGerber** [comment]
> Why not just use mod (num % 1) ?

This does not work with negative numbers.

**Comment by JakeGerber** [comment]
Anything else?

**Comment by JakeGerber** [comment]
@cclauss in case you missed it.

**Comment by cclauss** [comment]
Why not just use mod (num % 1) ?

**Comment by JakeGerber** [comment]
> Why not just use mod (num % 1) ? Or https://docs.python.org/3/library/functions.html#divmod

@cclauss once again moding does not work with negative numbers.

**Comment by cclauss** [comment]
All doctests use positive numbers.  Also, for completeness there should also be a test for zero.

https://github.com/TheAlgorithms/Python/pull/3700#issuecomment-716109561 Is not done.

**Comment by JakeGerber** [comment]
@cclauss I updated the doctests

**Review by cclauss — APPROVED** [review]
I learned about mod (%) and negative numbers.

**Pull request merged**